plotter/field_test.go

```diff
@@ -218,3 +218,44 @@ func ExampleField_gophers() {
 func TestFieldGophers(t *testing.T) {
 	cmpimg.CheckPlot(ExampleField_gophers, t, "gopher_field.png")
 }
+
+func TestFieldDims(t *testing.T) {
+	for _, test := range []struct {
+		rows int
+		cols int
+	}{
+		{rows: 1, cols: 2},
```

Comment:
perhaps also add `{rows:1, cols:1}` as well?
ditto for `HeatMap` ?

Reply:
That doesn't test anything else because of sequentiality of operations. So I have the two previously failing cases and a passing small case.